api/Navigator.json

```diff
@@ -1799,7 +1799,7 @@
               "notes": "Allowed schemes include <code>mailto</code>, <code>mms</code>, <code>nntp</code>, <code>rtsp</code>, and <code>webcal</code>. Custom protocols must be prefixed with <code>web+</code>."
             },
             "chrome_android": {
-              "version_added": true
+              "version_added": false
```

Comment:
Support of this feature in Chrome is a little more complicated.

https://www.chromestatus.com/features#registerProtocolHandler

I can help you work through this. Please start by mining the data in that link.

Reply:
Can you be more specific about what you mean here? When I tried `registerProtocolHandler` in Chrome for Android, it was `undefined` (plus I found [this open bug](https://bugs.chromium.org/p/chromium/issues/detail?id=178097)).

Reply:
I was not aware that it is not supported on Android. On desktop it requires a secure context [since Chrome 80](https://www.chromestatus.com/features/5756636801007616). [Since 77](https://www.chromestatus.com/features/5066079862784000) the URL argument only accepts 'http' or 'https'.